docs/howto/cylc-ui.md

```diff
@@ -1,60 +1,113 @@
 # Getting Started with Cylc 8
 
+## Development Installation
+
+> **Note**: We recommend developing in a Conda environment as this provides a
+  more standard setup though if you ensure the non-python dependencies
+  are installed correctly you can bypass this.
+
 1. Create a clean conda install:
   ```bash
-  conda create -n my-example-env python=3.7 --yes
+  conda create -n my-example-env python=3.7 configurable-http-proxy --yes
```

Comment:
Advise installing `configurable-http-proxy` via Conda.

No benefit to installing via yarn as we don't provide version pinning.